apidoc/Titanium/Media/Media.yml

```diff
@@ -401,12 +401,34 @@ methods:
         will crash if your app does not include the key. Check the [Apple docs](https://developer.apple.com/library/prerelease/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html)
         for more information.
     platforms: [iphone, ipad]
+    deprecated:
+        since: "6.1.0"
+        notes: Use <Titanium.Media.requestRecorderPermission> instead.
     since: 5.1.0
     parameters:
       - name: callback
         summary: Callback function to execute when the users responds to the authorization alert.
         type: Callback<MediaAuthorizationResponse>
 
+  - name: requestRecorderPermission
+    summary: Request the user's permission for audio recording.
+    description: |
+        On iOS 7.0, recording audio requires explicit permission from the user. After the user
+        grants or denies permission, the system remembers the choice for future use in the same app.
+        If permission is not granted, or if the user has not yet responded to the permission prompt,
+        any audio recording sessions record only silence.
+
+        In addition, Apple introduced the Info.plist key `NSMicrophoneUsageDescription` that is used to display an
+        own description while authorizing microphone permissions. In iOS 10, this key is mandatory and the application
+        will crash if your app does not include the key. Check the [Apple docs](https://developer.apple.com/library/prerelease/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CocoaKeys.html)
+        for more information.
+    platforms: [iphone, ipad]
```

Comment:
Wasn't the plan to align the permissions, so we have parity between all platforms here? Please do the Android part as well, before we go ahead.